packages/help-center/src/components/help-center.tsx

```diff
@@ -128,13 +128,23 @@ const HelpCenter: React.FC< Container > = ( { handleClose, hidden } ) => {
 
 	useSelect( ( select ) => ( select( USER_STORE ) as UserSelect ).getCurrentUser(), [] );
 
-	const currentSite = window?.helpCenterData?.currentSite;
+	/**
+	 * This site is provided by the backed in Editing Toolkit.
+	 * It's difficult to get the site information on the client side in Atomic sites. So we moved this challenge to the backend,
+	 * and forwarded the data using `localize_script` to the client side.
```

Comment:
Haha, it's weird that you use `wp_localize_script` to pass data that are not localization (@arcangelini in #64907?). Better use `wp_add_inline_script` for that.